lib/epoch.ex

```diff
@@ -1,27 +1,40 @@
 defmodule DateTimeParser.Epoch do
   @moduledoc false
 
-  import NimbleParsec
-  import DateTimeParser.Formatters, only: [format: 1]
-  import DateTimeParser.Combinators.Epoch
+  @token_key :unix_epoch
+  @max_subsecond_digits 6
 
-  defparsec(:parse, unix_epoch())
+  def parse(%{"seconds" => raw_seconds, "subseconds" => raw_subseconds}) do
```

Comment:
TIL named captures will still have the key with value `""` even if there's not a match on the key.